### ISSUE TITLE
Unable to run tests. Test discovery failed Visual Studio Mac

### ISSUE THREAD

**Issue by codito** [Microsoft/vstest#1115]
*Copied from https://github.com/Microsoft/testfx-docs/issues/18 on behalf of @danywalls*

I'm trying to run MsTest on but Visual Studio for Mac, Unit Test explorer detect my test but unnable run only show "Unable to run test. discovery failed.

**Comment by codito** [comment]
Tagging @DavidKarlas for help

**Comment by DavidKarlas** [comment]
MSTest support in Visual Studio for Mac will be available in 15.5 release cycle aka VSfM 7.3, which is planned to hit Alpha channel in few weeks.

**Comment by owenniblock** [comment]
If you're building tests for CI or you can run tests as part of a build script - you can use the `dotnet` cli on a mac (I'm using current stable build):

```
dotnet test CreditCardValidator.iOS.MSTestAppium/CreditCardValidator.iOS.MSTestAppium.csproj 
Build started, please wait...
Build completed.

Test run for /Users/owenniblock/test-apps/CCExampleAppium/CreditCardValidator.iOS.MSTestAppium/bin/Debug/netcoreapp2.0/CreditCardValidator.iOS.MSTestAppium.dll(.NETCoreApp,Version=v2.0)
Microsoft (R) Test Execution Command Line Tool Version 15.3.0-preview-20170628-02
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
Failed   CreditCardValidator.iOS.MSTestAppium.Tests.IntentionalFailure
Error Message:
 Assert.IsTrue failed. 
Stack Trace:
   at CreditCardValidator.iOS.MSTestAppium.Tests.IntentionalFailure() in /Users/owenniblock/test-apps/CCExampleAppium/CreditCardValidator.iOS.MSTestAppium/Tests.cs:line 21


Total tests: 3. Passed: 2. Failed: 1. Skipped: 0.
Test Run Failed.
Test execution time: 1.0736 Seconds
```
